src/fides/api/task/execute_request_tasks.py

```diff
@@ -315,10 +316,23 @@ def run_access_node(
         )
     except Exception as e:
         logger.error(f"Error in run_access_node: {e}")
+        # Mark the parent privacy request as errored so it is not re-dispatched.
+        try:
+            with self.get_new_session() as session:
+                pr = PrivacyRequest.get(db=session, object_id=privacy_request_id)
+                if pr:
+                    pr.status = PrivacyRequestStatus.error  # type: ignore[attr-defined]
+                    session.add(pr)
+                    session.commit()
+        except (
+            Exception
+        ) as db_exc:  # pragma: no cover – best effort to not mask original
+            logger.warning("Failed to mark privacy request errored: {}", db_exc)
```

Comment:
This is a leftover from my first pass. We should remove this since we have a more standard way of flagging a privacy request as errored using the `poll_for_exited_privacy_request_tasks` job